RLTSquare/CustomEmailSender/etc/frontend/di.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Framework\App\RouterList">
+        <arguments>
+            <argument name="routerList" xsi:type="array">
+                <item name="routingExample" xsi:type="array">
```

Comment:
Copy-paste - assuming this was copy paste and `name="routingExample"` sticked around. We should follow a certain naming convention.